sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/view/projects/ProjectView.java

```diff
@@ -137,6 +137,7 @@ private void selectProject(Project project) {
       removeFileDownloaders(downloadButton);
     }
     if (project.projectStatus().totalCount() < 1) {
+      samplesButton.setEnabled(false);
```

Comment:
why is this necessary? are such projects shown at all?

Reply:
yes they are, otherwise you can navigate to an empty samples view.